### ISSUE TITLE
Add Tron Chain in this app.

### ISSUE THREAD

**Issue by makhanait** [horizontalsystems/unstoppable-wallet-android#4268]
Hi, Please Add the Tron chain to the wallet. or can you provide some details on how we can add new blockchains in the wallet?

**Comment by esengulov** [comment]
yes, the number of supported blockchains keeps expanding. Latest addition was Polygon (MATIC)
